editor/plugins/shader_editor_plugin.cpp

```diff
@@ -655,6 +655,10 @@ ShaderEditorPlugin::ShaderEditorPlugin() {
 
 		menu_hb->add_child(make_floating);
 		window_wrapper->connect("window_visibility_changed", callable_mp(this, &ShaderEditorPlugin::_window_changed));
+	} else {
+		// Add placeholder button that displays tooltip about multi-window support being unavailable.
+		make_floating = memnew(ScreenSelect);
+		menu_hb->add_child(make_floating);
```

Comment:
Wouldn't it be better if you adjusted the above code instead? (like in the dock manager)
Right now the disabled button looks differently from non-disabled one.

Same in shader editor, where the button even changes position.